community/kernel/src/main/java/org/neo4j/kernel/api/security/AuthToken.java

```diff
@@ -48,6 +48,19 @@ static String safeCast( String key, Map<String,Object> authToken ) throws Invali
         return (String) value;
     }
 
+    static Map<String,Object> safeCastMap( String key, Map<String,Object> authToken )
```

Comment:
Is the variable name `authToken` really appropriate here? It seems to me that this method is used to extract a parameter map from a containing map, with no logical connection to auth tokens. I would suggest `map` or `parameterMap`.